pages/onboarding_cluster.ipynb

```diff
@@ -0,0 +1,156 @@
+    "To request access to one of these clusters please file a GitHub issue [here](https://github.com/operate-first/support/issues/new?assignees=first-operator&labels=kind%2Fonboarding%2Carea%2Fcluster&template=onboarding_to_cluster.yaml&title=NEW+PROJECT%3A+%3Cname%3E). \n",
+    "\n",
+    "\n",
+    "## Operation\n",
+    "\n",
+    "This part of tutorial servers as a guide for adding cluster scoped resources of `Namespace` and `Group` kinds. Following steps in this guide should result in a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository.\n",
+    "\n",
+    "### Prerequisites\n",
+    "Please fork/clone the [operate-first/apps](https://github.com/operate-first/apps) repository. During this whole setup, we’ll be working within this repository.\n",
+    "\n",
+    "For successful completion of this guide you need to understand what the aim is. Please have prepared following data:\n",
+    "- Name of the onboarded team.\n",
+    "- Desired namespace name. Please use your team name as a prefix. This will make it easier for you to onboard to ArgoCD later on.\n",
+    "- List of users you’d like to add to your team.\n",
+    "- An optional team GPG key, in case you would like to modify the encrypted list of users of your team later on.\n",
+    "\n",
+    "### Recipe\n",
+    "If you want to know more about the overall design please consult the ADR documentation at [operate-first/support](https://github.com/operate-first/blueprint).\n",
+    "In general we store all the cluster-scoped resources in a `cluster-scope` kustomize application within this repository.\n",
+    "#### Adding namespaces\n",
+    "For easier onboard to ArgoCD later on, we prefer to follow a name pattern for all our namespaces. Please use your team name as a prefix to the namespace name like so: `$OWNER_TEAM-example-project`.\n",
+    "#####  Base resources\n",
+    "\n",
+    "Please run:\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!opfcli create-project $NAMESPACE_NAME $OWNER_TEAM -d $OPTIONAL_PROJECT_DESCRIPTION"
+   ],
+   "outputs": [],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "This command will create:\n",
+    "\n",
+    "- A namespace in `cluster-scope/base/core/namespaces/$NAMESPACE_NAME`\n",
+    "- A blank user group for the `$OWNER_TEAM` if it does not exist yet in the `cluster-scope/base/user.openshift.io/groups/$OWNER_TEAM`\n",
+    "- An RBAC component for the project admin role `RoleBinding` in `cluster-scope/components/namespace-admin-rolebinding/$OWNER_TEAM` and maps it to the newly added namespace.\n",
+    "\n",
+    "##### Enabling namespace deployment to Zero cluster\n"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "!cd cluster-scope/overlays/prod/$ENV/$TARGET_CLUSTER"
```

Comment:
Undefined variables. Please add to the variables cell on the top.
```suggestion
    "!cd cluster-scope/overlays/prod/$TARGET_CLUSTER"
```

Reply:
Resolved in latest commit.